src/includes/index.ts

```diff
@@ -0,0 +1,26 @@
+import { combine, Store } from 'effector';
+
+export function includes <T extends string>(
+  a: Store<T>,
+  b: Store<T> | T,
+): Store<boolean>
+export function includes <T extends string | number>(
+  a: Store<Array<T>>,
+  b: Store<T> | T,
+): Store<boolean>
+export function includes <T extends string | number>(
+  a: Store<T | Array<T>>,
+  b: Store<T> | T,
```

Comment:
```suggestion
  haystack: Store<T | Array<T>>,
  needle: Store<T> | T,
```

Reply:
I think `a, b` is not clearly explains what in what we checking:

```ts
includes($email, "@");
includes("@", $email);
```

it will work by types in both ways, but IDE will suggest `a:`, `b:` names which is not self-descriptive, I think